pkg/cmd/util/filepath.go

```diff
@@ -47,3 +48,25 @@ func RelativizePaths(refs []*string, base string) error {
 	}
 	return nil
 }
+
+// RelativizePathWithNoBacksteps updates the given refs to be relative paths, relative to the given base directory as long as they do not require backsteps.
+// Any path requiring a backstep is left as-is
+func RelativizePathWithNoBacksteps(refs []*string, base string) error {
+	for _, ref := range refs {
+		// Don't relativize empty paths
+		if len(*ref) > 0 {
+			rel, err := filepath.Rel(base, *ref)
+			if err != nil {
+				return err
+			}
+
+			// if we have a backstep, don't mess with the path
+			if strings.HasPrefix(rel, "../") {
```

Comment:
> if we're going to use the original path, make sure it is absolute, otherwise we've changed the path

Updated along with the comment.  Test added to make sure it did what I thought it was going to do.